recipes/_lustre_install.rb

```diff
@@ -87,7 +87,7 @@
   end
 
   kernel_module 'lnet'
-elsif node['platform'] == 'amazon'
+elsif node['platform'] == 'amazon' and node['platform_version'].to_i == 1
```

Comment:
As you wrote some diff above
>       # Platform_version is a year rather than a version for the first major version of amazon linux.
so this ```node['platform_version'].to_i == 1``` would not work, since `platform_version.to_i` should be `2018`

Reply:
Ah yes, missed this.